source/input_viewer.cpp

```diff
@@ -136,14 +141,36 @@ void InputViewer::drawViewer(Vec2 pos, float scale, bool is_shadow) {
               25.f * scale);
 
     // analog sticks
-    drawStickOutline(is_shadow ? 0x00000060 : 0xFFFFFFFF,
+    uint32_t col;
+    double stickMag;
+    stickMag = tww_mPadStatus.stick_x*tww_mPadStatus.stick_x + tww_mPadStatus.stick_y*tww_mPadStatus.stick_y;
+    if ((g_tools[ESS_CHECKER_INDEX].active)) {
+        if (stickInRectRange(-2, -2, 2, 2)) {
+            col = 0x000000FF;
+        } else if (stickMag <= 81) {
+            col = 0x34DEEBFF;
+        } else if (stickMag <= 324) {
+            col = 0x40EB34FF;
+        } else if (stickMag <= 729) {
+            col = 0xF2E30FFF;
+        } else if (stickMag <= 1296) {
+            col = 0xDE9D12FF;
+        } else {
+            col = 0xB00E0EFF;
+        }
+    } else if (g_tools[DEADZONE_CHECKER_INDEX].active && stickInRectRange(-2, -2, 2, 2)) {
+        col = 0xFF0000FF;
+    } else {
+        col = 0xFFFFFFFF;
+    }
```

Comment:
If you don't mind, I made a color library to make it easier to re-use colors, so these colors should be added to it.

Reply:
This now uses the color library colors, I also fixed an error in color.h.